spec/ref_arch_setup/install_spec.rb

```diff
@@ -5,6 +5,15 @@
   let(:pe_conf_path)    { "/tmp/pe.conf" }
   let(:pe_tarball_path) { "/tmp/pe.tarball" }
   let(:install)         { RefArchSetup::Install.new(target_master) }
+  let(:task)            { "ref_arch_setup::install_pe" }
+  let(:params)          do
+    { "pe_conf_path" => pe_conf_path, "pe_tarball_path" => pe_tarball_path, \
+      "pe_target_master" => target_master }
```

Comment:
Why use strings for the keys instead of symbol? Seems like this should be:
    { pe_conf_path: pe_conf_path, pe_tarball_path: pe_tarball_path, \
      pe_target_master: target_master }

Reply:
Mainly cause of my perl background... But thinking more about it... The key ends up going to the command line as a string.  So maybe less converting is better?